AWG_Charter.md

```diff
@@ -15,7 +15,7 @@ The CVE Automation Working Group (AWG) was established to provide a forum for de
 
 Any active, CVE authorized program member may participate on the AWG. This includes [Board members](https://cve.mitre.org/community/board/index.html), [CVE Numbering Authority (CNA) representatives](https://cve.mitre.org/cve/cna.html), Authorized Data Publishers (ADP), and participants from the Secretariat&#39;s organization. In addition, the AWG is open to participation from the public at large. There is no limit to the number of representatives a given organization may have as members of the working group.
```

Comment:
Can these two links also be updated to https://www.cve.org/ProgramOrganization/Board and https://www.cve.org/PartnerInformation/Partner#CNA ?

Reply:
I have made these changes.